x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/indicator_match/threat_mapping/create_threat_signals.ts

```diff
@@ -216,6 +218,22 @@ export const createThreatSignals = async ({
     }
   };
 
+  const license = await firstValueFrom(licensing.license$);
+  const hasPlatinumLicense = license.hasAtLeast('platinum');
+  const isAlertSuppressionEnabled = Boolean(
+    completeRule.ruleParams.alertSuppression?.groupBy?.length
+  );
+
+  const isAlertSuppressionConfigured =
+    isAlertSuppressionEnabled &&
+    Boolean(runOpts.experimentalFeatures?.alertSuppressionForIndicatorMatchRuleEnabled) &&
+    hasPlatinumLicense;
+
+  // alert suppression needs to be performed on results searched in ascending order, so alert's suppression boundaries would be set correctly
+  // at the same time, there are concerns on performance of IM rule when sorting is set to asc, as it may lead to longer rule runs, since it will
+  // go through alerts that might ve been processed in earlier executions, when look back interval set to large values(it can't be larger than 24h)
```

Comment:
Adding a qualifier for clarity: 
```suggestion
  // first go through alerts that might ve been processed in earlier executions, when look back interval set to large values (it can't be larger than 24h)
```